client/src/pages/signUp.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import axios from 'axios'
+import Form from '../components/Form/Form'
+import '../scss/signup.css'
```

Comment:
Put your CSS in the the signup folder with signup.jsx file.
```suggestion
import './signup.css'
```